sdcm/argus_results.py

```diff
@@ -170,3 +170,20 @@ def _get_status_based_on_previous_results(metric: str):
     for key, value in stats.items():
         result_table.add_result(column=key, row="#1", value=value, status=_get_status_based_on_previous_results(key))
     argus_client.submit_results(result_table)
+
+
+def send_manager_benchmark_results_to_argus(argus_client: ArgusClient, result: dict, sut_timestamp: int,
+                                            row_name: str = None):
+    if not row_name:
+        row_name = "#1"
+
+    class RestoreResult(GenericResultTable):
+        class Meta:
+            name = f"Restore benchmark"
+            description = "Restore benchmark"
+            Columns = [ColumnMetadata(name=param, unit="", type=ResultType.FLOAT) for param in result.keys()]
```

Comment:
I would recommend no creating this class in runtime 
and clearly define the columns and their units 

the purpose of this class is that it would be visible and clear what exactly get reported

Reply:
Fixed